.github/workflows/job_onnx_models_tests.yml

```diff
@@ -0,0 +1,111 @@
+        uses: actions/download-artifact@v3
+        with:
+          name: openvino_package
+          path: ${{ env.INSTALL_DIR }}
+
+      - name: Download OpenVINO tests package
+        uses: actions/download-artifact@v3
+        with:
+          name: openvino_tests
+          path: ${{ env.INSTALL_TEST_DIR }}
+
+      # Needed as ${{ github.workspace }} is not working correctly when using Docker
+      - name: Setup Variables
+        run: |
+          echo "OPENVINO_REPO=$GITHUB_WORKSPACE/openvino" >> "$GITHUB_ENV"
+          echo "INSTALL_DIR=$GITHUB_WORKSPACE/install" >> "$GITHUB_ENV"
+          echo "INSTALL_TEST_DIR=$GITHUB_WORKSPACE/install/tests" >> "$GITHUB_ENV"
+
+      - name: Extract OpenVINO packages
+        run: |
+          pushd ${INSTALL_DIR}
+            tar -xzf openvino_package.tar.gz -C ${INSTALL_DIR}
+          popd
+
+          pushd ${INSTALL_TEST_DIR}
+            tar -xzf openvino_tests.tar.gz -C ${INSTALL_DIR}
+          popd
+
+      - name: Fetch setup_python action and model_zoo_preprocess script
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            .github/actions/setup_python/action.yml
+            src/frontends/onnx/tests/tests_python/model_zoo_preprocess.sh
+          sparse-checkout-cone-mode: false
+          path: 'openvino'
+
+      - name: Install dependencies
+        run: |
+          # install git (required to build pip deps from the sources)
+          apt-get update && apt-get install --assume-yes --no-install-recommends git ca-certificates git-lfs
+
+      - name: Setup Python 3.11
+        uses: ./openvino/.github/actions/setup_python
+        with:
+          version: '3.11'
+          should-setup-pip-paths: 'false'
+          self-hosted-runner: ${{ contains(inputs.runner, 'aks') }}
+
+      - name: Update Models
+        run: bash ${OPENVINO_REPO}/src/frontends/onnx/tests/tests_python/model_zoo_preprocess.sh -d ${MODELS_SHARE_PATH} -o -s "${{ env.ONNX_MODEL_ZOO_SHA }}"
+
+      - name: Install OpenVINO Python wheels
+        run: |
+          # Install the core OV wheel
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*.whl
+
+          extras_to_install="onnx,pytorch"
+
+          # Find and install OV dev wheel
+          pushd ${INSTALL_DIR}/tools
+            ov_dev_wheel_name=$(find . -name 'openvino_dev*.whl')
+            python3 -m pip install $ov_dev_wheel_name[$extras_to_install]
+          popd
+
+      - name: Install Python API tests dependencies
+        run: |
+          # To enable pytest parallel features
+          python3 -m pip install pytest-xdist[psutil] pytest-forked
+          # For torchvision to OpenVINO preprocessing converter
+          python3 -m pip install -r ${INSTALL_TEST_DIR}/python/preprocess/torchvision/requirements.txt
+
+          # TODO: replace with Python API tests requirements
+          python3 -m pip install -r ${INSTALL_TEST_DIR}/mo/requirements_dev.txt
```

Comment:
why do we need the following requirements ?
- openvino_dev
- torchvision/requirements.txt
- mo/requirements_dev.txt

Reply:
* openvino_dev - for the onnx installation and usage
* torchvision/requirements.txt, mo/requirements_dev.txt - they were mentioned in the https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/setup.cfg#L8 but for the base tests and not zoo_models, I'm unsure if they are required - I've removed them and will see how it goes.

Reply:
The tests seem to work: https://github.com/openvinotoolkit/openvino/actions/runs/7221174595/job/19676177446?pr=21581 - cancelled because the newer commit was pushed with rebased changes.